src/sentry/snuba/metrics/fields/base.py

```diff
@@ -701,6 +705,14 @@ class DerivedMetricKey(Enum):
             ),
             post_query_func=lambda *args: max(0, *args),
         ),
+        SingularEntityDerivedMetric(
+            metric_name=DerivedMetricKey.TRANSACTION_ALL.value,
+            metrics=[TransactionMetricsKey.TRANSACTION.value],
```

Comment:
We actually do not have a metric called `transaction`. There is a metric called `transaction.duration` to which you can apply `countIf` to get the total number of transactions:

https://github.com/getsentry/relay/blob/4b7bfc0c38c0e78056054e8b5d4b66c78c4798f5/relay-server/src/metrics_extraction/transactions.rs#L318